confignode/src/assembly/resources/conf/iotdb-confignode.properties

```diff
@@ -57,6 +53,23 @@ config_node_rpc_port=22277
 # Datatype: int
 # thrift_init_buffer_size=1024
 
+####################
+### consensus protocol configuration
+####################
+
+# ConfigNodeGroup consensus protocol type
+# These consensus protocol are currently supported:
+# 1. standalone(No protocol, only supports stand-alone machine)
+# 2. ratis(Ratis protocol)
+# Datatype: String
+# consensus_type=standalone
```

Comment:
Default is Ratis?

Reply:
Set default to standalone will be easier for develop. I suppose that change this default value after our development is almost completed will be better.